src/components/study-container.js

```diff
@@ -304,6 +306,35 @@ export function StudyContainer({ view, onChangeTab }) {
         }
     }, [studyUuid, loadTree]);
 
+    useEffect(() => {
+        const appName = 'GridStudy';
+
+        if (studyUuid) {
+            fetchElementAndParentsInfo(studyUuid)
+                .then((response) => {
+                    const study = response[0];
+                    const parents = response
+                        .slice(1)
+                        .map((parent) => parent.elementName);
+
+                    document.title = computePageTitle(appName, study, parents);
+                })
+                .catch((errorMessage) => {
+                    document.title = appName;
+                    displayErrorMessageWithSnackbar({
+                        errorMessage: errorMessage,
+                        enqueueSnackbar: enqueueSnackbar,
+                        headerMessage: {
+                            headerMessageId: 'LoadStudyAndParentsInfoError',
+                            intlRef: intlRef,
+                        },
+                    });
+                });
+        } else {
+            document.title = appName;
```

Comment:
maybe it's useless ? title is already set to something like 'GridStudy' elsewhere

Reply:
it's useful when studyUuid change then ok